site/content/en/docs/tasks/run/argo_workload.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Run An Argo Workflow"
+date: 2024-01-03
+weight: 7
+description: >
+  Integrate Kueue with Argo Workflows.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Argo Workflows](https://argo-workflows.readthedocs.io/en/latest/).
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+As of the writing of this doc, Kueue doesn't support Argo Workflows [Workflow](https://argo-workflows.readthedocs.io/en/latest/workflow-concepts/) resources directly,
+but you can take advantage of the ability for Kueue to [manage plain pods](/docs/tasks/run_plain_pods) to integrate them.
+
+1. By default, the integration for `v1/pod` is not enabled.
+   Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+   and enable the `pod` integration.
+
+   The pod integration for Argo Workflows looks like:
+   ```yaml
+   apiVersion: config.kueue.x-k8s.io/v1beta1
+   kind: Configuration
+   integrations:
+     frameworks:
+      - "pod"
+     podOptions:
+       # You can change namespaceSelector to define in which
+       # namespaces kueue will manage the Workflow pods.
+       namespaceSelector:
+         matchExpressions:
+         - key: kubernetes.io/metadata.name
+           operator: NotIn
+           values: [ kube-system, kueue-system ]
+       # Argo Workflows uses the workflows.argoproj.io/completed label to
+       # keep track of pods it manages. We will use that as a hint for Kueue
+       # to find Argo Workflow pods.
+       podSelector:
+         matchExpressions:
+         - key: workflows.argoproj.io/completed
+           operator: In
+           values: [ "false" ]
+   ```
+
+2. Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration.
+   For example, pods managed by `batch/v1.Job` won't be managed by `pod` integration.
+
+4. Check [Administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial Kueue setup.
+
+## Workflow definition
+
+When creating [Workflow](https://argo-workflows.readthedocs.io/en/latest/workflow-concepts/),
+[WorkflowTemplate](https://argo-workflows.readthedocs.io/en/latest/workflow-templates/),
+[ClusterWorkflowTemplate](https://argo-workflows.readthedocs.io/en/latest/cluster-workflow-templates/),
+or [CronWorkflow](https://argo-workflows.readthedocs.io/en/latest/cron-workflows/) resources for
+Kueue, take into consideration the following aspects:
+
+### a. Targeting a single LocalQueue
+
+If you want the entire workflow to target a single [local queue](/docs/concepts/local_queue),
+it should be specified in the `spec.podMetadata` section of the Workflow configuration.
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
```

Comment:
please more the examples to https://github.com/kubernetes-sigs/kueue/tree/main/site/static/examples/jobs for consistency as a single source of examples easy to run.

Reply:
done.